src/main/java/TaskList.java

```diff
@@ -0,0 +1,188 @@
+import java.io.*;
+import java.util.ArrayList;
+
+public class TaskList {
```

Comment:
Perhaps deleteTask, markAsDone and markAsNotDone could return something so that it would be easier to do tests on the class later on and printing of statements could be handled by another class